lib/config/hse_gparams.c

```diff
@@ -62,6 +61,26 @@ bool HSE_NONNULL(1, 2, 3)
     return true;
 }
 
+static merr_t
+logging_destination_stringify(
+    const struct param_spec *const ps,
+    const void *const              value,
+    char *const                    buf,
+    const size_t                   buf_sz,
+    size_t *const                  needed_sz)
+{
+    static const char *values[] = { "stdout", "stderr", "file", "syslog" };
+
+    const int n = snprintf(buf, buf_sz, "\"%s\"", values[*(enum log_destination *)value]);
```

Comment:
For safety, should verify the index into the `values` array.  I would make it a live check (rather than assert or INVARIANT) since it's nowhere near data path code.

Reply:
It would be a developer error for the value never to be a correct index because the incorrect value never would have been set. I will throw an assert in there.